featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -898,3 +925,20 @@ def strip_values_if_series(values):
     if isinstance(values, pd.Series):
         values = values.values
     return values
+
+
+def is_agg_type_in_primitive_or_feature(f):
```

Comment:
I'm not sure if this is a relevant comment since this might be out of the scope of this issue, and the rest of the file uses f as the standard iteration variable for feature, but as a newbie I feel a more descriptive variable name would be helpful for understanding.

Reply:
Yah we can change this to `feat` or `feature` for the parameter argument name